app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -18,6 +18,12 @@
   transition: $transition-properties;
 }
 
+@mixin transform($transform-properties...) {
```

Comment:
Ok, so I logged an issue to look into this further (https://github.com/forever-inc/forever-style-guide/issues/80), and ultimately it seems like adding autoprefixer here is the right move. But we should handle that outside this already monstrous PR.